style.css

```diff
@@ -0,0 +1,137 @@
+html,
+body {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  background: #e5e5e5;
+}
+
+header {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  right: 0;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 54px;
+  background: #fff;
+}
```

Comment:
It will be nice if we remove the fixed width `width: 375px;` to allow the design responsiveness from (375px to 768px) 🤓👍

Your design
![image](https://github.com/ethred/Portfolio/assets/78251927/d8ba7f4e-5b65-46ce-b834-9fbe74549e72)